v1.1/connection-parameters.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Client Connection Parameters
+summary: This page describes the parameters used to establish a client connection.
+toc: false
+---
+
+Client applications, including client [`cockroach`
+commands](cockroach-commands.html), work by establishing a network
+connection to a CockroachDB cluster. The client connection parameters
+determine which CockroachDB cluster they connect to, and how to
+establish this network connection.
+
+<div id="toc"></div>
+
+
+## Supported Connection Parameters
+
+There are two principal ways a client can connect to CockroachDB:
+
+- Most client apps, including most `cockroach` commands, use a SQL connection
+  established via a [PostgreSQL connection URL](#connect-using-a-url). When using a URL,
+  a client can also specify SSL/TLS settings and additional SQL-level parameters. This mode provides the most configuration flexibility.
+- Most `cockroach` commands also provide [discrete connection parameters](#connect-using-discrete-parameters) that
+  can specify the connection parameters separately from a URL. This mode is somewhat less flexible than using a URL.
+- Some `cockroach` commands support connections using either a URL
+  connection string or discrete parameters, whereas some only support
+  discrete connection parameters.
+
+The following table summarizes which command supports which connection parameters:
+
+Client | Supports [connection by URL](#connect-using-a-url) | Supports [discrete connection parameters](#connect-using-discrete-parameters)
+-------|----------------------------|-----------------------------------
+Client apps using a PostgreSQL driver | ✓ | (✗)
+[`cockroach init`](initialize-a-cluster.html) | ✗ | ✓
+[`cockroach quit`](stop-a-node.html) | ✗ | ✓
+[`cockroach sql`](use-the-built-in-sql-client.html) | ✓ | ✓
+[`cockroach user`](create-and-manage-users.html) | ✓ | ✓
+[`cockroach zone`](configure-replication-zones.html) | ✓ | ✓
+[`cockroach node`](view-node-details.html) | ✓ | ✓
+[`cockroach dump`](sql-dump.html) | ✓ | ✓
+[`cockroach debug zip`](debug-zip.html) | ✗ | ✓
+
+## Connect Using a URL
+
+SQL clients, including some [`cockroach` commands](cockroach-commands.html) can connect using a URL.
+
+A connection URL has the following format:
+
+~~~
+postgres://<username>:<password>@<host>:<port>/<database>?<parameters>
+~~~
+
+Component | Description
```

Comment:
Instead of putting `optional` in the description, I'd suggest a third column, `Required`, using the checks and xs to indicate whether required or not.